nifi-docs/src/main/asciidoc/user-guide.adoc

```diff
@@ -577,10 +577,8 @@ The first configuration option is the Scheduling Strategy. There are three possi
 |1-12 or JAN-DEC
 
 |Day of Week
-|1-7 or SUN-SAT
+|0-6 or SUN-SAT (0 or 7 is SUN)
```

Comment:
It's a bit odd that the range is 0-6 but 7 = SUN? Is there a better way of expressing this?

Reply:
Thanks for highlighting that example, I changed the documentation to read `0-7` with `(0 or 7 is Sunday)`. That follows the Cron Expression documentation from the Spring Framework, and other places. Open to other suggestions if you think additional clarification is needed.